app/components/toggle.hbs

```diff
@@ -35,4 +40,8 @@
       </svg>
     </span>
   </span>
+
+  {{#if (and @tooltipCopy (not-eq @tooltipCopy ""))}}
+    <EmberTooltip @tooltipClass="normal-case font-normal" @text={{@tooltipCopy}} />
```

Comment:
@libmartinito why do we have to update `@tooltipClass` here? I don't remember doing this elsewhere

Reply:
If this is to avoid outer classes messing with styling, then the correct fix is to instead change outer classes to not override styling (we can't possible account for the 100s of different text styles outer classes might apply - what if they override `color` tomorrow? or `text-decoration`?).